specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/common/2022-01-15-preview/parameters.json

```diff
@@ -0,0 +1,62 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-01-15-preview",
+    "title": "Common types and definitions",
+    "description": "Definitions and parameters common for all resource types of KubernetesConfiguration RP"
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {},
+  "parameters": {
+    "ClusterRpParameter": {
```

Comment:
>ClusterRpParameter [](http://example.com/codeflow?start=5&length=18)

This parameter had its x-ms-enum extension removed (previously had modelAsString: true). This removal isn't an ARM issue, but might be an SDK issue.

Reply:
I am not seeing any difference between my parameters and the ones from previous versions. Can you provide more detail on what this is referring to?